quiz/envs/backend.default.env

```diff
@@ -1,3 +1,4 @@
 DJ_SECRET_KEY=
 DJ_DEBUG=
 DJ_ALLOWED_HOSTS=
+LOGGING_LVL=
```

Comment:
Do README.md przydałaby się instrukcja jak niektóre zmienne mają być wypełnione.
Bądź jeśli to możliwe to default bym wrzucił do *.default.env

* jak wygenerować secret key
* jaką wartość dać do debuga
* jaką wartość dać do allowed_hosts
* jakie wartości można dać do logging_lvl

Ten sam problme dla postgres.env
Już widziałem z 2 osoby co nie wypełniły envów tylko skopiowały